src/raglite/_database.py

```diff
@@ -270,28 +273,41 @@ class Eval(SQLModel, table=True):
     document: Document = Relationship(back_populates="evals")
 
     @staticmethod
-    def from_chunks(
-        question: str, contexts: list[Chunk], ground_truth: str, **kwargs: Any
+    def from_contexts(
```

Comment:
Would call it `from_context` (singular) for concistency with `context: list[ChunkSpan]` arguments elsewhere.